lib/components/molecules/AtomicCarousel/AtomicCarousel.js

```diff
@@ -0,0 +1,341 @@
+
+  /**
+   * Rendering list of dots
+   *
+   * @param {Object()} setItem
+   *   Using to set the slide number.
+   * @param {number} currentItem
+   *   Show current slide number.
+   */
+  function getSliderDots(currentItem, setItem) {
+    const countElements = [];
+
+    for (let id = 1; id <= slideCount; id += 1) {
+      countElements.push(
+        <li key={id} className={classnames('atomic-item', { active: id === currentItem })}>
+          <Button
+            type="button"
+            onClick={() => specificNode(id, setItem)}
+            onKeyUp={handlePreviousKey}
+            className="icon"
+          >
+            {id}
+          </Button>
+        </li>
+      );
+    }
+
+    return countElements;
+  }
+
+  /**
+   * Rendering wrapper of dots
+   *
+   * @param {Object()} setItem
+   *   Using to set the slide number.
+   * @param {number} currentItem
+   *   Show current slide number.
+   *
+   * @return HTML
+   *   Wrapper with UL | OL
+   */
+  const getList = (setItem, currentItem) => {
+    return (
+      <div className="atomic-dots">
+        <ListVanilla>{getSliderDots(currentItem, setItem)}</ListVanilla>
+      </div>
+    );
+  };
+
+  /**
+   * Wrapper of slide.
+   *
+   * @param {Object} slide
+   *   the slide content that need to be wrapped.
+   */
+  const slideWrapper = (slide, id = 1) => (
+    <div key={id} className="slide-item">
+      {slide}
+    </div>
+  );
+
+  /**
+   * Show required number of items on single slide.
+   *
+   * @param {Array} content
+   *   Its the children data.
+   * @param {Number} perFrame
+   *   No. of item needs to show in single slide.
+   * @param {Number} currentScreen
+   *   Its the current slide number.
+   *
+   * @return Array
+   *   Number of slides per frame.
+   */
+  const getPerFrameItems = (content, perFrame, currentScreen) => {
+    const itemsArray = [];
+    const start = currentScreen === 1 ? 0 : perFrame * (currentScreen - 1);
+
+    if (perSlideFrame > 1) {
+      for (let i = start; i <= currentScreen * perFrame - 1; i += 1) {
+        itemsArray.push(slideWrapper(content[i], i));
+      }
+    } else {
+      itemsArray.push(slideWrapper(content[currentScreen - 1]));
+    }
+
+    return itemsArray;
+  };
+
+  /**
+   * Handling next key event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleNextKey = event => {
+    if (event.key === 'ArrowRight') {
+      handleNext();
+    }
+  };
+
+  /**
+   * Handling Mouse Enter event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleMouseEnter = () => {
+    if (mouseStatus) {
+      setAnimation(false);
+    }
+  };
+
+  /**
+   * Handling Mouse Leave event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleMouseLeave = () => {
+    if (mouseStatus) {
+      setAnimation(true);
+    }
+  };
+
+  /**
+   * Slider Section.
+   */
+  const getSliderSection = () => {
+    return (
+      <div className="carousel-container">
```

Comment:
What is there from accessibility? How will a disabled user would know its a carousel and its autoplay ect etc